katana/wapps/execution/templates/execution/execution_settings.json

```diff
@@ -0,0 +1 @@
+{"defaults": {"start_dir": "default", "jira_proj": "project3"}, "jira_proj": [{"name": "warrior"}, {"name": "project1"}, {"name": "project2"}, {"name": "project3"}]}
```

Comment:
Why added default Jira project (warrior) is added ?

Reply:
Made necessary changes and committed.